packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -41,13 +41,25 @@ $Rating-largeIconSize: 20px;
 
     color: $Rating-defaultColor;
 
+    @media screen and (-ms-high-contrast: active) {
+      color: WindowText;
+    }
+
     &.starIsSelected {
       color: $Rating-selectedColor;
+
+      @media screen and (-ms-high-contrast: active) {
+        color: Highlight;
```

Comment:
You are correct @jahnp, window and highlight are backgrounds. Windowtext and highlighttext are foregrounds.